recipes/sciplot/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: sciplot
+  version: {{ version }}
+
+source:
+  # url: https://github.com/sciplot/sciplot/archive/v{{ version }}.tar.gz
+  # sha256: d40508a78d0aa88dbbbe35efdbd056fca971cf4d2eff4a1707f69c34e4018fad
+  url: https://github.com/allanleal/sciplot/archive/fix-for-conda-forge.tar.gz
```

Comment:
Can you apply these as patches in the original source?

Reply:
Ops, I forgot to revert to `url: https://github.com/sciplot/sciplot/archive/v{{ version }}.tar.gz`, instead of using the branch `fix-for-conda-forge.tar.gz`.